test/hls/hls_parser_unit.js

```diff
@@ -1160,6 +1160,72 @@ describe('HlsParser', () => {
     await testHlsParser(master, media, manifest);
   });
 
+  it('when characteristics are empty then add \n',
```

Comment:
This doesn't look right.  it() doesn't take multiple string arguments.  If you need to continue the string on the next line, please do something like this:

```js
it('first part of the line ' +
    'second part of the line', async () => {
```

Reply:
Acknowledged. Will resolve soon.

Reply:
Edited and fixed myself.